test/run-openshift

```diff
@@ -17,6 +17,18 @@ test -n "${IMAGE_NAME-}" || false 'make sure $IMAGE_NAME is defined'
 test -n "${VERSION-}" || false 'make sure $VERSION is defined'
 test -n "${OS-}" || false 'make sure $OS is defined'
 
+function assert_cmd_fails() {
+  local result=""
+
+  eval "$@" &>/dev/null || result=1
+  if [ "$result" = "1" ]; then
+    echo " PASS"
+    return 0
```

Comment:
Slightly more readable version:
```bash
if eval "$@" &>/dev/null ; then
   echo " FAIL"
   return 1
fi
echo " PASS"
```

Reply:
Added your version of assert_cmd_fails. Thanks for the suggestion!